docs/tutorial/2-tilt-up.md

```diff
@@ -36,48 +36,47 @@ tilt up --port=3366
 
 Built-in functions like [`k8s_yaml`][api-k8s_yaml] and [`docker_build`][api-docker_build] register information with the Tilt engine.
-At the end of the execution, Tilt uses the resulting configuration to assemble resources to build and deploy.
+At the end of the execution, Tilt uses the resulting configuration to assemble resources to build and deploy. Then, based on the assembled resources, Tilt watches your source code files so it can trigger a rebuild on the associated service(s). 
+
+Later on, we'll explore how Tilt makes it possible to optimize this process even more. You can skip container re-builds and Pod re-deployments entirely via [Smart Rebuilds with Live Update][tutorial-live-update].
 
-Additionally, based on the assembled resources, Tilt watches your source code files to trigger a rebuild on the associated service(s).
-Later, in the [Smart Rebuilds with Live Update][tutorial-live-update] section, we'll explore how Tilt makes it possible to optimize this process even more to skip container re-builds and Pod re-deployments entirely!  
+(If you're curious, go ahead and open the [`tilt-avatars` Tiltfile][repo-tilt-avatars-tiltfile] and read through it.
+We won't tell anyone you peeked.)
 
 For now, that's all you need to know!
 <!-- TODO(milas): snarky graphic about how that ^^^ was a galaxy brain info dump? -->
 
-If you're curious, go ahead and open the [`tilt-avatars` Tiltfile][repo-tilt-avatars-tiltfile] and read through it.
-We won't tell anyone you peeked.
-
-## Resources
-A "resource" is a bundle of work managed by Tilt such as a Docker image to build + Kubernetes YAML to apply.
+## On Resources
+A "resource" is a bundle of work managed by Tilt. For example: a Docker image to build + a Kubernetes YAML to apply.
 
 > 😶‍🌫️ **Resources don't have to be containers!**
 >
 > Tilt can also [manage locally-executed commands][local-resource] to provide a unified experience no matter how your code runs.  
 
 Resource bundling is automatic in most cases: Tilt finds the relationship between bits of work (e.g. `docker_build` + `kubectl apply`).
-When that's not sufficient, `Tiltfile` functions like [`k8s_resource`][api-k8s_resource] let you configure resources on top of Tilt's automatic assembly.
+When that's not sufficient, `Tiltfile` functions like [`k8s_resource`][api-k8s_resource] let you configure resources on top of what Tilt does automatically.
 
-Because Tilt assembles multiple bits of work into a single resource, it's much easier to determine status and find errors across build, deploy, and runtime!
+Because Tilt assembles multiple bits of work into a single resource, it's much easier to determine status and find errors across build, deploy, and runtime.
 
 ### Update Status
-Whenever you run `tilt up` or change a source file, Tilt determines which resources need to be "updated".
+Whenever you run `tilt up` or change a source file, Tilt determines which resources need to be "updated." 
```

Comment:
This convention of putting the period inside the quotes often feels wrong but I think it's APA style?

Reply:
You're right! Apparently outside the quotes is British-style while inside is American-style? TIL

Reply:
I do look forward to continue doing this "wrong" moving forward because there's no way I will remember/adapt 😬